sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/metrics/MetricsPublisher.java

```diff
@@ -26,9 +26,9 @@
   public static void toConsole(PipelineResult result, String namespace) {
```

Comment:
Similar comment to above: please document the recipe for how this is intended to be used. Presumably this would fail unless the the monitor transformations were correctly applied to collect metrics. 

It would be sufficient to link to an example test pipeline.

Reply:
ok